components/elm/cime_config/testdefs/testmods_dirs/elm/fates_cold_twostream/shell_commands

```diff
@@ -0,0 +1,15 @@
+module load e4s
+spack env activate gcc
+spack load nco
+
+SRCDIR=`./xmlquery SRCROOT --value`
+CASEDIR=`./xmlquery CASEROOT --value`
+FATESDIR=$SRCDIR/components/elm/src/external_models/fates
+FATESPARAMFILE=$CASEDIR/fates_params_twostream.nc
+
+ncgen -o $FATESPARAMFILE $FATESDIR/parameter_files/fates_params_default.cdl
+
+$FATESDIR/tools/modify_fates_paramfile.py --O --fin $FATESPARAMFILE --fout $FATESPARAMFILE --var fates_rad_model --val 2 --allpfts
```

Comment:
Why is the parameter file generated on the fly by `ncgen` and then modified, instead of having the file added to the inutdeck server?

Reply:
We update the parameter file frequently in FATES. This prevents us from having to make changes to multiple test files every time we change an irrelevant parameter (which would also be a liability to injecting bugs.. ie what if we don't update all the files correctly every time there is a change).

Note that in the future, it is intended that we will have many of these on-the-fly parameter files generated, so that we can stress test the model with many different configurations.  It would not be a matter of updating 1 or two files, but think more like 5-10.